modules/ROOT/pages/configure-service-mesh.adoc

```diff
@@ -9,6 +9,7 @@ image::configure-service-mesh-breadcrumb.png[]
 The following topics are covered in this section:
 
 * Provision Service Mesh Adapter
+* Create a non-Mule application
```

Comment:
Lets not call it a non-mule application or API. Please change this

Create an API in Anypoint Platform

Reply:
Done!